### PR TITLE
Added bluesound documentation

### PR DESCRIPTION
**Pull request by thrawnarn**
**Description:**
Documentation for the bluesound component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7192

### CODE REVIEW

File: source/_components/media_player.bluesound.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+The `bluesound` platform allows you to control your [Bluesound](http://www.bluesound.com/) HiFi wireless speakers and audio components from Home Assistant.
+
+If you want to automatically discover new devices, just make sure you have discovery: in your configuration.yaml file. To manually add a Bluesound device to your installation, add the following to your configuration.yaml file:
+
+
+```yaml
+# Example configuration.yaml entry with manually specified addresses
+media_player:
```
Comment (Landrash) [inline source/_components/media_player.bluesound.markdown]: Generalize the examples to have the minimal amount of configuration needed.
Feel free to add a advanced example if you feel it's necessary below.

**Comment by balloob** [comment]
PR in main branch went stale.

**Comment by rowco** [comment]
Any chance of this being resurrected..

**Comment by thrawnarn** [comment]
Yes I will fix this as soon as possible. I haven't had any time available to look into this. Hopefully I will have time over in a few weeks. hold tight :)

**Comment by rowco** [comment]
That would be amazing, thanks.

**Pull request merged**